src/hierarchies/likelihoods/laplace_likelihood.cc

```diff
@@ -0,0 +1,53 @@
+#include "laplace_likelihood.h"
+
+double LaplaceLikelihood::compute_lpdf(const Eigen::RowVectorXd &datum) const {
+  return stan::math::double_exponential_lpdf(
+      datum(0), state.mean, stan::math::sqrt(state.var / 2.0));
+}
+
+void LaplaceLikelihood::update_sum_stats(const Eigen::RowVectorXd &datum,
+                                         bool add) {
+  if (add) {
+    // sum_abs_diff_curr += std::abs(state.mean - datum(0, 0));
+    cluster_data_values.push_back(datum);
```

Comment:
I would do it as in the FALikelihood where we only use the index set of the data currently allocated to this cluster and the dataset_ptr

Reply:
(so basically this would be just an empty method)